cmd/libs/go2idl/generator/import_tracker.go

```diff
@@ -44,48 +49,110 @@ func NewImportTracker(types ...*types.Type) *ImportTracker {
 			// Add other forbidden keywords that also happen to be
 			// package names here.
 		},
+		lang: "golang",
 	}
 	tracker.AddTypes(types...)
 	return tracker
 }
 
+func NewProtobufImportTracker(local types.Name, types ...*types.Type) *ImportTracker {
```

Comment:
That will not solve the problem, because there is some proto-specific code in AddType() and ImportLines() now.
So we would need to duplicate most of the code anyway (if we don't want to have it here...).

>
I agree protobuf is pretty fundamental. But I don't think go2idl is about Kube - it's a generic tool. The other specific things around it also won't be specific to kube, but to anything building similar machinery.

I agree, but this change is not making this code kube-specific, right?